### ISSUE TITLE
Color the sim cards based on the color from the settings

### ISSUE THREAD

**Issue by derrohrbach** [SimpleMobileTools/Simple-Dialer#322]
It would be nice, if the sim card color would be respected for the call buttons and in the call log. If you are interested in this feature, then i might even make a pull request for it. Also interesting for the SMS app.

![grafik](https://user-images.githubusercontent.com/13820214/159303517-a4f5dfe7-04dd-4984-8cc8-f4a136963f77.png)

**Comment by tibbi** [comment]
Are you settings that color in your device settings or where? Does any other app show it?

**Comment by derrohrbach** [comment]
Exactly, the setting is in the device settings and it is used by multiple apps to make it easier to differenciate between different sim cards.
- pin input screen
- AOSP Dialer
- Google Dialer
- AOSP SMS App
- Google Messages

Maybe it is this? Just me googling for five minutes. I am not really experienced with android APIs.
https://developer.android.com/reference/android/telecom/PhoneAccount#getHighlightColor()

If you want i can also send screenshots of the Google Dialer, have to activate it again though :D

**Comment by leonp5** [comment]
@derrohrbach would be nice if you can add this feature :-)
